manifests/m/Microsoft/VisualStudio/Locator/3.0.3.9285/Microsoft.VisualStudio.Locator.locale.en-US.yaml

```
@@ -0,0 +1,15 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Microsoft.VisualStudio.Locator
+PackageVersion: 3.0.3.9285
+PackageLocale: en-US
+Publisher: Microsoft Corporation
+PackageName: Visual Studio Locator
+License: MIT
+Copyright: Copyright (C) Microsoft Corporation. All rights reserved.
+Description: Find and get information on Visual Studio versions 2017 and newer with optional additional criteria such as version, channel, or installed workloads.
+ShortDescription: Find Visual Studio installations
+ManifestType: defaultLocale
+ManifestVersion: 1.1.0
```

```suggestion
ManifestVersion: 1.2.0
```
`1.2.0` schema includes support for `InstallerType: portable`